examples/src/main/java/com/mapbox/navigation/examples/ui/CustomCameraActivity.kt

```diff
@@ -6,7 +6,6 @@ import com.mapbox.api.directions.v5.models.BannerInstructions
 import com.mapbox.api.directions.v5.models.DirectionsRoute
 import com.mapbox.mapboxsdk.location.modes.RenderMode
 import com.mapbox.mapboxsdk.maps.MapboxMap
-import com.mapbox.navigation.base.extensions.ifNonNull
```

Comment:
Other possible fix

import com.mapbox.navigation.utils.extensions.ifNonNull

Reply:
Up to you, both work 🚀